src/codegen/codegen_ispc_visitor.cpp

```diff
@@ -25,6 +25,19 @@ using symtab::syminfo::Status;
 
 using visitor::RenameVisitor;
 
+const std::vector<ast::AstNodeType> CodegenIspcVisitor::incompatible_node_types = {
+    ast::AstNodeType::VERBATIM,
+    ast::AstNodeType::EIGEN_NEWTON_SOLVER_BLOCK,
+    ast::AstNodeType::EIGEN_LINEAR_SOLVER_BLOCK,
+    ast::AstNodeType::WATCH_STATEMENT,
+    ast::AstNodeType::TABLE_STATEMENT};
+
+const std::unordered_set<std::string> CodegenIspcVisitor::incompatible_var_names = {
+    "cdo",           "cfor",           "cif",    "cwhile",   "foreach",      "foreach_active",
+    "foreach_tiled", "foreach_unique", "in",     "noinline", "__vectorcall", "int8",
+    "int16",         "int32",          "int64",  "launch",   "print",        "soa",
+    "sync",          "task",           "varying"};
+
```

Comment:
This seems strange formatting to me. we should disable clang-format for this block ?

Reply:
I find it quite nice instead!

Reply:
`beauty lies in the eye of the beholder` - I am fine with it then :)